test/ng/directive/ngModelSpec.js

```diff
@@ -455,6 +455,42 @@ describe('ngModel', function() {
     });
 
 
+    describe('$setModelValue', function() {
+
+      it('should set the value to $modelValue', function() {
+        ctrl.$setModelValue(10);
+        expect(ctrl.$modelValue).toBe(10);
+      });
+
+      it('should update the value on the scope', inject(function($compile) {
```

Comment:
Hmm, I guess you are right.
If we had `ngModel.$formatModel`, then I suspect we would need to have a complementary `ngModel.$parseView`? Since changes to the formatters might well include changes to the parsers too? In which case one would want the new viewValue to be parsed and update the model....